korge/src/commonMain/kotlin/com/soywiz/korge/ui/UIScrollableArea.kt

```diff
@@ -0,0 +1,108 @@
+package com.soywiz.korge.ui
+
+import com.soywiz.korge.html.*
+import com.soywiz.korge.view.*
+
+inline fun Container.uiScrollableArea(
+	width: Number = 256.0,
+	height: Number = 256.0,
+	contentWidth: Number = 512.0,
+	contentHeight: Number = 512.0,
+	buttonSize: Number = 32.0,
+	verticalScroll: Boolean = true,
+	horizontalScroll: Boolean = true,
+	font: Html.FontFace = defaultUIFont,
+	skin: UISkin = defaultUISkin,
+	upSkin: UISkin? = null,
+	downSkin: UISkin? = null,
+	leftSkin: UISkin? = null,
+	rightSkin: UISkin? = null,
```

Comment:
Ditto. Why not use a single UISkin and allow there to provide specific skins defaulting to the basic ones? This would allow to simplifying the propagation of the skin. Maybe you can use a typed mechanism like the coroutines context to provide ways to extend the available skin stuff and provide method/property extensions in this file like:
```fun UISkin.withScrollLeftButton(...): UiSkin```

Reply:
What about creating additional data class ScrollableAreaSkin that will include these properties with default values: skin, upSkin, downSkin, leftSkin, rightSkin?